src/Seq.Input.RabbitMQ/RabbitMQInput.cs

```diff
@@ -44,6 +50,12 @@ public class RabbitMQInput : SeqApp, IPublishJson, IDisposable
             HelpText = "The RabbitMQ queue name to receive events from. The default is `Logs`.")]
         public string RabbitMQQueue { get; set; } = "logs";
 
+        [SeqAppSetting(
+            DisplayName = "Ssl",
+            IsOptional = true,
+            HelpText = "Whether or not the connection is with ssl. The default is false.")]
```

Comment:
SSL should be capitalized in the help text.

Reply:
changed